src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts

```diff
@@ -174,8 +173,7 @@ export class BackLayerWebView<T extends ICommonCellInfo> extends Themable {
 		super(themeService);
 
 		const logsPath = joinPath(environmentService.windowLogsPath, 'notebook.rendering.log');
-		this._renderLogger = this._register(loggerService.createLogger(logsPath, { name: 'notebook.rendering' }));
-		registerLogChannel(logChannelId, nls.localize('renderChannelName', "Notebook rendering"), logsPath, fileService, logService);
+		this._renderLogger = this._register(loggerService.createLogger(logsPath, { id: logChannelId, name: nls.localize('renderChannelName', "Notebook rendering") }));
```

Comment:
@rebornix you do not need to create output channel now. You can include all necessary info while creating the logger and system will create it automatically.

Reply:
Sorry wrong ping. CC @mjbvz